newscoop/library/Newscoop/Services/TemplatesService.php

```diff
@@ -66,5 +130,13 @@ private function preconfigureSmarty()
 
         // reverse templates dir order
         $this->smarty->setTemplateDir(array_reverse($this->smarty->getTemplateDir()));
+
+        $publicationMetadata = $this->publicationService->getPublicationMetadata();
+        if (count($publicationMetadata) > 0) {
+            $this->smarty->campsiteVector = array(
```

Comment:
Not in this PR. There will be next one will improvements for deployments - and it will be special parameter prepended to vector 'params'.